storage/src/nodestore/mod.rs

```diff
@@ -832,7 +832,7 @@ mod tests {
         value: Some(vec![9, 10, 11].into_boxed_slice()),
         children: from_fn(|i| {
             if i == 15 {
-                Some(Child::AddressWithHash(nonzero!(1u64), std::array::from_fn::<u8, 32, _>(|i| i as u8).into()))
+                Some(Child::AddressWithHash(LinearAddress::new(1).unwrap(), std::array::from_fn::<u8, 32, _>(|i| i as u8).into()))
```

Comment:
we could potentially create our own macro

```rust
macro_rules linear_addr {
    ($v:literal) => {const { $crate::alloc::LinearAddress::new($v).unwrap() }};
}
```

Which will evaluate the literal at compile time.

Reply:
Yes, this is a good idea. I don't know why the nonzero crate doesn't do that.